tests/cpp/engine/threaded_engine_test.cc

```diff
@@ -258,47 +258,50 @@ TEST(Engine, PushFunc) {
 TEST(Engine, PushFuncND) {
   auto ctx = mxnet::Context{};
   mxnet::NDArray nd(ctx);
+  std::vector<mxnet::NDArray*> nds;
+  nds.push_back(&nd);
+  void** pnds = reinterpret_cast<void**>(nds.data());
```

Comment:
We might want some tests for cases with more than 1 NDArray present in the argument call.

Reply:
Hi @KellenSunderland, I have added the testcase with more than 1 NDArray : )